samples/radio_test/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  name: Radio Test
+  description: Radio Test sample
+tests:
+  test_build:
+    build_only: true
+    build_on_all: true
+    platform_whitelist: nrf52_pca10040 nrf52840_pca10056 nrf5340_dk_nrf5340_cpunet
```

Comment:
What about nrf52833_pca10100 ? Is it supported ? There are appeals to `NRF52833_XXAA` in code.

Reply:
Thanks for pointing it out. I removed ```NRF52833_XXAA``` because this SoC is not supported in NCS yet.